charts/wildfly/values.yaml

```diff
@@ -110,7 +110,7 @@ wildfly:
   ## @param wildflyPassword WildFly user password
   ## Defaults to a random 10-character alphanumeric string if not set
   ##
-  wildflyPassword: "test"
+  wildflyPassword: "mestest1234"
```

Comment:
Our CI setup doesn't allow to modify more than one chart at the time.

```bash
git checkout origin/master charts/wildfly/values.yaml
```

Reply:
Yes exactly. @yvesjaggi could you please make 2 PRs, one for i2b2, one for wildfly?

Reply:
outdated